structlog_sentry/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import sys
+
+from sentry_sdk import capture_exception, capture_message
+
+
+class SentryProcessor:
+    """Sentry processor for structlog. Uses Sentry SDK to capture events in Sentry."""
+
+    def __init__(self, sentry_log_level=logging.ERROR):
+        self.sentry_log_level = sentry_log_level
+
+    def _log(self, event_dict, level="error"):
```

Comment:
There is no need to have default level. It should be always passed (match event level). This may lead to bug, when someone will forget to pass it and everything will be turned into errors.